src/main/java/com/twilio/oai/api/RubyApiResourceBuilder.java

```diff
@@ -222,7 +223,7 @@ private void updateContextResourceDependents(Map<String, String> dependentsForOp
         if(listObjs != null) {
             for (DirectoryStructureService.ContextResource cr : listObjs) {
                  String value = (dependentsForOperation.containsKey(cr.getFilename())) ? dependentsForOperation.get(cr.getFilename())
```

Comment:
Do we not need to change this line also? since we are modifying the mapOperationsDependents to always have `resource_url` as key?

Reply:
There are some use cases where dependentsForOperation.get(cr.getFileName()) has some data. How it is populated, will have to dig in further. So, as to not break the existing flow, I have kept it here.

Reply:
The values in dependentProperties would be of different formats like abc.json from file name and v1/something/abc via path, will that work fine? Can you add some testcases too? it'd be easier to understand the behaviour.

Reply:
yes, after this change, filename is no longer used to set mapping. Hence, removed it.